File: rust/raindrops.rs
pub fn raindrops(n: u32) -> String {
    let mut droplet: String = String::new();
    if n % 3 == 0 {
        droplet += "Pling";
    }
    if n % 5 == 0 {
        droplet += "Plang";
    }
    if n % 7 == 0 {
        droplet += "Plong";
    }
    if droplet.is_empty() {
        return format!("{}", n.to_string());
    }
    droplet
}

File: clojure/raindrops.clj
(ns raindrops)

(defn convert [x]
  (let [res (str (if (zero? (mod x 3)) "Pling")
                 (if (zero? (mod x 5)) "Plang")
                 (if (zero? (mod x 7)) "Plong"))]
    (if (empty? res)
      (str x)
      res)))